appinventor/components/src/com/google/appinventor/components/runtime/Sprite.java

```diff
@@ -697,7 +752,12 @@ protected int hitEdge() {
    * canvas. If the sprite is too tall to fit on the canvas, this aligns the
    * top side of the sprite with the top side of the canvas.
    */
-  @SimpleFunction
+  @SimpleFunction(
```

Comment:
Technically only public methods should be marked `@SimpleFunction`. There's a public version `MoveIntoBounds` on line 664 that covers this requirement, so I think you can kill this annotation.

Reply:
You're right. Done.